### DISCUSSION TITLE
How can I enable "Optional addons" later ?

### DISCUSSION THREAD

**Discussion by sim-san** [nextcloud/all-in-one#1143] [Questions] [answered]
**What I have done**

1. Stop the containers
2. check the checkboxes for the addons i want to add
3. Start the containers

**What happend**
The marks get removed after the start of the containers
I do not see that there are containers added

**Comment by Zoey2936** [comment, marked as answer]
![grafik](https://user-images.githubusercontent.com/75573284/189690050-84e77e40-18db-4bb3-b2cf-10a7b2ae72ca.png)
Did you press the "Save changes" Button?

**Reply by sim-san** [reply]
Thanks
